samples/Calling/src/app/utils/AppUtils.ts

```diff
@@ -57,10 +57,14 @@ export const isOnIphoneAndNotSafari = (): boolean => {
   return userAgent.includes('iPhone') && (userAgent.includes('FxiOS') || userAgent.includes('CriOS'));
 };
 
-export const isMobileSession = (): boolean =>
-  !!window.navigator.userAgent.match(/(iPad|iPhone|iPod|Android|webOS|BlackBerry|Windows Phone)/g);
+/*
+ * TODO:
+ *  Remove once landscape check is
+ */
+// export const isMobileSession = (): boolean =>
+//   !!window.navigator.userAgent.match(/(iPad|iPhone|iPod|Android|webOS|BlackBerry|Windows Phone)/g);
```

Comment:
just completely remove these all this commented code and associated todo in code.
If we want to show an error in landscape/too small we should create a bug but I think its unecessary as we document this in our documentation.

Reply:
As a general pattern we observe in this repo we delete blocks of commented code. We can always look in the git history for how a function looked. 

In rare occasions if its a small snippet that helps readability we do maintain some commented lines but that is generally more rare.

Reply:
can't we just say .. isLandscape = (): boolean => window.innerWidth < window.innerHeight and use that?

Reply:
In addition, we can use it with isMobileSession to check if the user is on mobile and on landscape. I will give that a go and test it out.